integrations/dav-server/src/lib.rs

```diff
@@ -0,0 +1,23 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+mod webdav_dir_entry;
+mod webdav_file;
+mod webdav_metadata;
+pub mod webdavfs;
+
+pub use dav_server::DavHandler;
```

Comment:
We don't need to expose `DavHandler`, it's users decision. `dav-server-opendalfs` should expose an `OpendalFs` so that users can use them like:

```rust
use dav_server::{fakels::FakeLs, DavHandler};
use dav_server_opendalfs::OpendalFs;

let dav_server = DavHandler::builder()
    .filesystem(OpendalFs::new(op))
    .locksystem(FakeLs::new())
    .build_handler();
```

Reply:
> use dav_server::{fakels::FakeLs, DavHandler};
use dav_server_opendalfs::OpendalFs;

user should import `dav_server` and `dav_server_opendalfs`  together in `Cargo.toml` ?

Reply:
> user should import `dav_server` and `dav_server_opendalfs` together in `Cargo.toml` ?

Yep.

Reply:
The ultimate goal of this project is to merge with `dav-server`. Imagine that we are the maintainers of `dav-server`. How should we organize this project?